regression/win32-regression-master.bat

```diff
@@ -15,17 +15,24 @@ rem midnight.
 rem Consider using date-based log file names.
 rem c:\myscript.%date:~-4%%date:~4,2%%date:~7,2%.%time::=%.log 2>&1
 
-set logdir=e:\regress\logs
+set logdir=c:\regress\logs
 
 rem Ensure Git/bin/sh.exe is not in the PATH.  It interferes with mingw operations.
-set PATH=%PATH:c:\Program Files\Git\bin;=%
+rem set PATH=%PATH:c:\Program Files\Git\bin;=%
+set PATH=%PATH:C:\Users\107638\AppData\Local\Programs\Git\bin\;=%
+
+rem Fix LANL proxy issues
+C:\Users\107638\AppData\Local\Programs\Git\bin\git.exe config --global http.proxy http://proxyout.lanl.gov:8080
+rem Don't set these because they prevent posting to rtt.lanl.gov/cdash.
+rem set HTTP_PROXY=http://proxyout.lanl.gov:8080
+rem set HTTPS_PROXY=http://proxyout.lanl.gov:8080
 
 rem Change '/c' to '/k' to keep the command window open after these commands 
 rem finish.
 
-%comspec% /c ""e:\regress\draco\regression\update_regression_dir.bat"" > %logdir%\update_regression_dir.log 2>&1
-%comspec% /c ""e:\regress\draco\regression\win32-regress.bat"" > %logdir%\regression-master.log 2>&1
+%comspec% /c ""c:\regress\draco\regression\update_regression_dir.bat"" > %logdir%\update_regression_dir.log 2>&1
```

Comment:
Why is the drive changed - more space now?

Reply:
I'm moving to a new machine that only has 1 drive instead of 2.